sentry_sdk/integrations/ray.py

```diff
@@ -58,18 +60,23 @@ def _f(*f_args, _tracing=None, **f_kwargs):
             """
             _check_sentry_initialized()
 
+            root_span_name = qualname_from_function(f) or DEFAULT_TRANSACTION_NAME
+            sentry_sdk.get_current_scope().set_transaction_name(
+                root_span_name,
+                source=TRANSACTION_SOURCE_TASK,
+            )
```

Comment:
Does this work before we even started the transaction? Maybe logically it'd better fit inside the `root_span` context manager?

Reply:
Because this sets the transaction name only on the scope, this should be fine. 

The other question this brought up inside me is why the ray integration does no isolation scope forking anywhere... 

I will tests this tomorrow.

Reply:
Ok, this works and is correct.